spec/casbin/model/model_spec.rb

```diff
@@ -1,339 +0,0 @@
-# frozen_string_literal: true
```

Comment:
Why do we delete this tests?

Reply:
overlapped by tests from spec/casbin/enforcer_spec.rb
And doesn't work with stubs